suite-native/receive/src/components/DeviceScreen/DeviceScreenContent.tsx

```diff
@@ -154,7 +170,17 @@ export const DeviceScreenContent = ({
                     <DeviceScreenPagination
                         deviceModel={deviceModel}
                         activePage={activePage}
-                        isAddressRevealed={isAddressRevealed}
+                        isPaginationShown={isAddressShown}
+                    />
+                )}
+                {loadingString && (
```

Comment:
Please create component for this instead of hook and use component directly here. You will also avoid unneccasrry rerenders of whole canvas with every "dot" change.

Reply:
9fe8f1e341